src/flb_input_chunk.c

```diff
@@ -602,9 +602,9 @@ int flb_input_chunk_find_space_new_data(struct flb_input_chunk *ic,
             FS_CHUNK_SIZE_DEBUG_MOD(o_ins, old_ic, -old_ic_bytes);
             o_ins->fs_chunks_size -= old_ic_bytes;
 
-            flb_debug("[input chunk] remove route of chunk %s with size %ld bytes to output plugin %s "
-                      "to place the incoming data with size %ld bytes", flb_input_chunk_get_name(old_ic),
-                      old_ic_bytes, o_ins->name, chunk_size);
+            flb_info("[input chunk] remove route of chunk %s with size %zd bytes to output plugin %s "
```

Comment:
the format specifiers here were wrong according to my understanding. 

`old_ic_bytes` is `ssize_t` which should use `%zd` for signed type, and then the others are `size_t` which is unsigned and needs `%zu`

Reply:
In terms of changing from info => debug, my thinking was that the user should get a clear message when chunks are being deleted.

Reply:
if I am not wrong the debug message is a "hint" to tell the user/debugger the chunk "aims to be removed", actually that routine is trying to find a candidate since the queue is full, then the logic depends on routes/mask, at that level of the message is not really removed. 

cc: @leonardo-albertovich @pwhelan -> I think you both were involved in those changes, can you comment ?

Reply:
Ok, in that case then I think the debug message on line 624 should be info and should be updated since that is where the deletion truly happens? And then here the debug message should say something like "Attempt to remove".

Reply:
flb_task_destroy also destroys the chunk backing the task. You would need to change the call to flb_debug on both lines 616 and 624.

Reply:
Ack. I will update to that.

Reply:
@pwhelan @edsiper I updated the existing debug message to be "consider removal" and then made a nice new info message in both places where we actually delete a chunk. Please re-review.